Cesium3DTilesSelection/include/Cesium3DTilesSelection/Tileset.h

```diff
@@ -494,6 +494,10 @@ class CESIUM3DTILESSELECTION_API Tileset final {
   // scratch variable so that it can allocate only when growing bigger.
   std::vector<const TileOcclusionRendererProxy*> _childOcclusionProxies;
 
+  std::shared_ptr<CesiumGeospatial::Ellipsoid> _ellipsoid;
+
+  const CesiumGeospatial::Ellipsoid& GetEllipsoid() const;
```

Comment:
To match our naming conventions:

```suggestion
  const CesiumGeospatial::Ellipsoid& getEllipsoid() const;
```